server/randomSentencesServer.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express')
+
+const app = express()
+const port = 3000
+const clientDirPath = "C:\\Dev\\Learning\\sentences\\client"
+
+const greateSentences = [
```

Comment:
I would put this in another files, and export a function called `getRandomGreatSentencs`, that way you separate the content logic from the routing logic. This methodology is also called model-view-controller (the model is responsible for data, and the controller for delivering it to the view)